workspace/__ardulib__/Graphics/Text.h

```diff
@@ -0,0 +1,38 @@
+
+#ifndef TEXT_H
+#define TEXT_H
+
+#include "XGraphics.h"
+#include "Fonts/glcdfont.h"
+
+using xod::XString;
+
+/*
+ * Represents a text given by a pivot point (x, y) which is the top left point of the text.
+ * By now this class uses only the standard 5x7 GLCD ASCII font.
+ * The `setTextColor` method sets up text color otherwise
+ * the default foreground color of the scene is used.
+ * The `setTextScale` method sets up the value that scales the font.
+ */
+class Text : public XGraphics {
+private:
+    XVector2<int16_t> _pivot;
+
+    XString* _str = nullptr;
```

Comment:
No need to use pointers for `XString`. It’s a thin scalar wrapper that is safe to pass by value. If not set it iterates as an empty string.